bin/bugsnag-maze-runner

```diff
@@ -23,13 +33,38 @@ class MazeRunnerEntry
       opt :init, 'Initialises a new Maze Runner project'
 
       opt :version, 'Display Maze Runner and Cucumber versions'
-      opt OPTION_SEPARATE_SESSIONS, 'Start a new Appium session for each scenario', short: :none
+      opt 'separate-sessions', 'Start a new Appium session for each scenario',
```

Comment:
Looks like this got changed from `OPTION_SEPARATE_SESSIONS`

```suggestion
      opt OPTION_SEPARATE_SESSIONS, 'Start a new Appium session for each scenario',
```

Reply:
Yep, not sure what I did there.